docs-source/content/release-notes.md

```diff
@@ -8,7 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| TBD | v2.2 | no  | TBD 
+| TBD | v2.2 | no  | Added support for FMW Infrastructure domains. If servers are manually shutdown, for example using the WLS Console, the operator will return them to the desired state specified in the domain custom resource.  Operator is now built and runs on JDK 11. 
```

Comment:
@markxnelson, we didn't agree (yet) to include the change to the scripts to restart servers that had been manually stopped.  There should be a way to restart such servers (and there is if you include deleting the pod), but having the server automatically restart under you if you've manually shutdown seems wrong.

Reply:
Operator now supports the graceful shutdown of server instances.